src/documentation/docs/components/Layout.mdx

```diff
@@ -0,0 +1,29 @@
+import Code from "@/components/internal/Documentation/Code";
+import ExampleLoader from "@/documentation/utils/ExampleLoader";
+import Meta from "@/components/internal/Documentation/Meta";
+import PropsTable from "@/components/internal/Documentation/PropsTable";
+import PropsTableRow from "@/components/internal/Documentation/PropsTable/components/PropsTableRow";
+
+
+<Meta
+  title="Layout / LayoutItem "
+  subtitle="A LayoutItem can be used inside of an Layout to render various content"
+/>
+
+##Properties
```

Comment:
These are only the properties for `Layout`. Please add the properties for `LayoutItem` as well.